internal/report/report_operator_templates.go

```diff
@@ -0,0 +1,28 @@
+package report
+
+const (
+	operatorTextReportTemplate = `
+Operator Install Report
+-----------------------------------------
+Report Date: {{ now }}
+OpenShift Version: {{ .OcpVersion }}
+Package Name: {{ .Subscription.Package }}
+Channel: {{ .Subscription.Channel }}
+Catalog Source: {{ .Subscription.CatalogSource }}
+Install Mode: {{ .Subscription.InstallModeType }}
+Result: {{ if .CsvTimeout }}timeout{{ else }}{{ .Csv.Status.Phase }}
+Message: {{ .Csv.Status.Message }}
+Reason: {{ .Csv.Status.Reason }}
+{{ end }}
+-----------------------------------------
+`
+	operatorJsonReportTemplate = `
+{
```

Comment:
Same thing about upload.

Reply:
oh does this just need to not use backticks?

Reply:
or rather you need to do
"string" +
"string2" +
"string3"

and so on?

Reply:
I'm questioning whether we even need to bother with this? Will this be edited often (unlikely)? Is this for human consumption? (Not really).

I'd say leave this alone on a single line. If you want to make it easier to edit in the future, maybe it wouldn't be a bad idea to put this version in a comment above it, and use that as a template, then make it clear that it needs to remain like this?

Reply:
That sounds good to me :)